index.js

```diff
@@ -112,26 +142,24 @@ class Dynamodb extends Datastore {
      * @param  {Object}   config.params      Record data
      * @param  {String}   config.params.id   Unique id. Typically the desired primary key
      * @param  {Object}   config.params.data The data to update with
-     * @param  {Function} callback           fn(err, data)
-     *                                       err - Error object
-     *                                       data - Data saved in the table
+     * @return {Promise}                     Resolves to the record that was updated
      */
-    _update(config, callback) {
+    _update(config) {
         const id = config.params.id;
         const userData = config.params.data;
         const client = this.clients[config.table];
         const updateOptions = {
             expected: { id }
         };
 
-        if (!client) {
-            return callback(null, null);
-        }
+        return new Promise((resolve, reject) => {
+            if (!client) {
+                return resolve(null);
```

Comment:
That's true. I'm not sure why. This is just translating from the old code. I feel like it should rejects here too... Maybe it was a bug.